# error
Test(read={'foo':2}) -> Test(write={'foo':2})
